modules/app_people.rb

```diff
@@ -0,0 +1,49 @@
+require './student'
+require './teacher'
+
+module People
+  def list_all_people(people)
+    puts 'No person record created yet' if people.empty?
+    people.map { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+  end
+
+  def create_student(age, name, people)
+    print 'Has parent permission to? [Y/N]: '
+    permission = gets.chomp
+    yes_answers = %w[y yes]
+    no_answers = %w[n no]
+    if yes_answers.include? permission.downcase
+      student = Student.new(age, name, true)
+    elsif no_answers.include? permission.downcase
+      student = Student.new(age, name, false)
+    else
```

Comment:
- [ ] in this statement you are repeating the same code twice this is not according to [the DRY principle ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid)please change it so complies with the DRY rules.
- [ ] in this method you are printing to the console, getting data from the user and saving the data as well, this is not in line with the single responsibility rules, please have another look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more details.